### DISCUSSION TITLE
OasisMist 1000S, but Canadian model (LUH-M101S-WCA)

### DISCUSSION THREAD

**Discussion by yeah-mike** [webdjoe/pyvesync#210] [Devices and Features]
Hi,

I see that support for the LUH-M101S-WUS was recently added. Amazon.ca has this listed, but with a Canadian model number LUH-M101S-W**CA**. Would this library support it? Or would it need update/modifications?

**Comment by webdjoe** [comment]
Thank you for the request, this will be in the next release 2.1.13
